pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -64,17 +66,30 @@ func qemuSystemProgram() (string, error) {
 
 func qemuFirmwarePath() (string, error) {
 	arch := runtime.GOARCH
+	// For macOS, find the correct brew installation path for qemu firmware
+	if runtime.GOOS == "darwin" {
+		p := "/usr/local/Cellar/qemu"
```

Comment:
I would probably used a switch here and added "amd64" here (like in the Linux version), but ok

Reply:
good call, switch added